{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "0b4c3417_9d53c178",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1060831
      },
      "writtenOn": "2021-12-13T10:02:06Z",
      "side": 1,
      "message": "Should we also change development/gsi/gsi_util/gsi_util/checkers/sepolicy_checker.py to complain if such a file exists?\n\n(If so, would be good to have a bug to tie the changes together.)",
      "revId": "2ef6693572efaac6422223cf90b474efa758d88b",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}